cl/opinion_page/templates/court.html

```diff
@@ -82,71 +64,37 @@ <h1>Decisions for the Tennessee Court of Workers' Compensation Claims and Tennes
             </span>
           </div>
           <div class="clearfix"></div>
-          <div class="input-group">
-            <div class="sidebar-checkbox inline-block">
-              <label for="id_tennworkcompcl">
-                <input id="id_tennworkcompcl" name="court_tennworkcompcl" checked="checked" type="checkbox"> Include CWCC Cases
-              </label>
-            </div>
-            <div class="sidebar-checkbox inline-block">
-              <label for="id_tennworkcompapp">
-                <input id="id_tennworkcompapp" name="court_tennworkcompapp" checked="checked" type="checkbox"> Include Appeals Board Cases
-              </label>
-            </div>
-          </div>
         </div>
       </form>
     </div>
-
-
     <div class="row v-offset-above-6">
-      <div class="col-xs-4">
-        <h3>Recent CWCC&nbsp;Decisions&nbsp;<i class="gray fa fa-file-text-o"></i></h3>
-        {% if request.user|in_group:"tenn_work_uploaders" %}
-          <p><a href="{% url "court_publish_page" pk="tennworkcompcl" %}"
-                class="btn btn-default">Upload New CWCC Opinion</a></p>
-        {% endif %}
-        {% if results_compcl.paginator.count > 0 %}
-          {% include "includes/search_result.html" with results=results_compcl type_override=SEARCH_TYPES.OPINION simple=True %}
-          <p class="right">
-            <a href="/?court_tennworkcompcl=on&order_by=dateFiled+desc"
-               class="btn btn-default btn-lg v-offset-above-2"
-               rel="nofollow"
-            >See Recent Opinions</a>
-          </p>
-        {% else %}
-          <p>No cases yet.</p>
-        {% endif %}
-      </div>
-      <div class="col-xs-4">
-        <h3>Recent Appeals Board&nbsp;Decisions&nbsp;<i class="gray fa fa-file-text-o"></i></h3>
-        {% if request.user|in_group:"tenn_work_uploaders" %}
-          <p><a href="{% url "court_publish_page" pk="tennworkcompapp" %}"
-                class="btn btn-default">Upload New Appeals Board Opinion</a></p>
+      <div class="col-xs-8">
+        <h3>Recent Decisions of the&nbsp;{{ court }} <i class="gray fa fa-file-text-o"></i></h3>
+        {% if request.user|in_group:"uploaders_{{ pk }}" %}
+          <p><a href="{% url "court_publish_page" pk="me" %}"
```

Comment:
Should the pk here be dynamic? Wouldn't this load the me upload page for everybody (including tenn?)

Reply:
yes